Header header
float32 left_setpoint
float32 right_setpoint

int64 left_enc_pos
int64 right_enc_pos
float32 left_enc_speed
float32 right_enc_speed

bool left_bumper
bool right_bumper
